src/bundle/Resources/public/js/scripts/fieldType/ezdate.js

```diff
@@ -73,10 +73,13 @@
             return;
         }
 
+        const selectedDateWithUserTimezone = eZ.helpers.timezone.convertDateToTimezone(date[0]);
```

Comment:
Just a question: what about passing `userTimezone` to convertDateToTimezone, like in `ezdatetime`?
https://github.com/ezsystems/ezplatform-admin-ui/blob/329b69b980572ab96a726223901fba0094f9bf0b/src/bundle/Resources/public/js/scripts/fieldType/ezdatetime.js#L81
this won't work?

Reply:
yes, it could be changed as it is in ezdatetime, i push changed, now

Reply:
If it works, then great!